alpaca/alpaca_test.go

```diff
@@ -265,6 +266,106 @@ func (s *AlpacaTestSuite) TestAlpaca() {
 		assert.Nil(s.T(), actualLastTrade)
 	}
 
+	// get latest trade
```

Comment:
I'd prefer to move this to a separate test case but agree with you to stay consistent with the rest of the file

Reply:
Yeah, I did not want to change things in this PR